compiler/passes/parallel.cpp

```diff
@@ -1037,12 +1037,10 @@ freeHeapAllocatedVars(Vec<Symbol*> heapAllocatedVars) {
 //  CHPL_COMM == "gasnet" && CHPL_GASNET_SEGMENT == "everything";
 // or
 //  CHPL_STACK_CHECKS == 0 && CHPL_TASKS == "fifo"
-// or
-//  CHPL_STACK_CHECKS == 0 && CHPL_TASKS == "muxed"
 //
 // true otherwise.
 //
-// The tasking layer matters because fifo and muxed allocate
+// The tasking layer matters because fifo allocates
```

Comment:
I have a PR on a branch that updates this text and code for qthreads -- I'l include this fix there